dev-lang/go/go-9999.ebuild

```diff
@@ -123,17 +127,30 @@ src_compile()
 	export GOHOSTARCH=$(go_arch ${CBUILD})
 	export GOHOSTOS=$(go_os ${CBUILD})
 	export CC=$(tc-getBUILD_CC)
+	export GOOS=$(go_os)
+
+	cd src
+
+	# Build the target support first.
```

Comment:
ah, sorry, missed that. So that's an artefact of make.bash being a bit to simple for our case but it deletes go_bootstrap so we cannot just add the call to build std for the second arch.